frontend/src/app/api/basicErrorController.service.ts

```diff
@@ -31,15 +33,24 @@ export class BasicErrorControllerService {
     protected basePath = 'https://test.h2ms.org:81';
     public defaultHeaders = new HttpHeaders();
     public configuration = new Configuration();
+    config: Config;
 
-    constructor(protected httpClient: HttpClient, @Optional()@Inject(BASE_PATH) basePath: string, @Optional() configuration: Configuration) {
+    constructor(protected httpClient: HttpClient, @Optional()@Inject(BASE_PATH) basePath: string, @Optional() configuration: Configuration,
+                @Optional() configService: ConfigService) {
         if (basePath) {
             this.basePath = basePath;
         }
         if (configuration) {
             this.configuration = configuration;
             this.basePath = basePath || configuration.basePath || this.basePath;
         }
+
+        this.config = configService.getConfig();
```

Comment:
Could cause a Null pointer exception. It looks like this occurs in all of the API code, so ditto for the other files.